### PR TITLE
Add CUAVA-2 and WS-1 satellites

### PR DESCRIPTION
**Pull request by patrickoppel**
As a representative of the University of Sydney team, I'd like to add the CUAVA-2
and Waratah Seed-1 CubeSats. 

Please let me know if this is not enough for an endorsement of
commercial satellites.

### CODE REVIEW

**Review by daniestevez — CHANGES_REQUESTED** [review]
Thanks. This looks good. Enough endorsement, since the pull request comes from a team representative.

Can you add a newline at the end of each SatYAML file, add an alternative name for WS-1 and squash into a single commit?

The syntax for the alternative name would be something like
```
alternative_names:
  - Waratah Seed-1
```
usually placed under `name:`.

**Comment by patrickoppel** [comment]
Added new line, alternative_name and squashed

File: python/satyaml/WS-1.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: WS-1
+alternative_names: Waratah Seed-1
```
Comment (daniestevez) [inline python/satyaml/WS-1.yml:2]:
```suggestion
alternative_names:
  - Waratah Seed-1
```

**Comment by daniestevez** [comment]
Looks good now. Merging.

**Pull request merged**